apps/errant-tower-client/src/pages/auth/auth-page.module.scss

```
@@ -69,6 +70,31 @@
+    mask-image: radial-gradient(ellipse 60% 80% at center, black 25%, transparent 60%);
+    -moz-mask-image: radial-gradient(ellipse 60% 80% at center, black 25%, transparent 60%);
+
+    img {
+        opacity: 0.9;
+        width: 90%;
+        animation: colorRotate 12s;
+        animation-iteration-count: infinite;
+        animation-direction: alternate;
+    }
+
+    @keyframes colorRotate {
```

Stylelint (17.9.0)

[error] Expected keyframe name "colorRotate" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)